geemap/timelapse.py

```diff
@@ -1286,6 +1286,92 @@ def naip_timelapse(
         raise Exception(e)
 
 
+from datetime import date
```

Comment:
I would suggest putting Lines 1289-1303 inside the `sentinel1_timeseries(roi=None)` function. Like this:

```python
from datetime import date
CURRENT_YEAR = date.today().year
if roi is None:
  roi = (
              [
                  [
                      [-115.471773, 35.892718],
                      [-115.471773, 36.409454],
                      [-114.271283, 36.409454],
                      [-114.271283, 35.892718],
                      [-115.471773, 35.892718],
                  ]
              ],
              None,
              False,
          )

```